src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php

```diff
@@ -54,8 +54,8 @@ public function getConfigurationFields()
             array(
                 "name" => "url",
                 "type" => "text",
-                "label" => gettext("URL"),
-                "help" => gettext("The Base URL to Nextcloud without trailing slash. For example: https://cloud.example.com"),
+                "label" => gettext("Nextcloud Server URL"),
+                "help" => gettext("The Base URL to Nextcloud without trailing slash. For example: https://cloud.example.com If your server SSL certificate is not issued from a well known authority, you will need to add your intermediate certificate to the OPNSense CA truststore to avoid SSL verification errors."),
```

Comment:
This one is basic network knowlege and expected from users of OPNsense to know (that certificates are validated and how). I would not like that but I don't care if it is merged as well.

Reply:
There's a (no more) surprising number of people having no clue about those things though, so IMO it's better to have it there than not. Saves some grey hairs ;)

Reply:
Again here: documentation can be added and expended. We do not need to overdo GUI labels and help texts.